osama-breakout-room/ArvinAlhissou/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html >
+
+<head>
+    <meta charset="UTF-8">
+    <title>twitter page</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <script src="https://kit.fontawesome.com/9016f6ba0a.js"></script>
+</head>
+
+<body>
+
+    <header>
+        <div class="wrapper">
+
+            <div class="logo">
+                <img src="csslogo.png" alt="csslogo">
+            </div>
+
+            <nav>
+                <a class="btn btn-bordered " href="#">
+                    LOGIN
+                </a>
+                <a class="btn btn-blue" href="#">
+                    REGISTER NEW ACCOUNT
+                </a>
+            </nav>
+            
+        </div>
+    </header>
+
+    <div class="wrapper">
```

Comment:
Awesome work on using the wrapper class to give a fixed width and centering your webpage content. 👍